app/controllers/projects_controller.rb

```diff
@@ -57,10 +57,16 @@ def create_project
       CreateProject.call(project_params: project_params, current_user: current_user)
   end
 
+  def update_project
+    @update_project ||=
+      UpdateProject.call(project_params: project_params, current_user: current_user, project_id: @project.id)
```

Comment:
You shouldn't pass the `project_id` to the interactor and then search it there. Instead you can just pass `project` into it.